.azure-pipelines/client.yml

```diff
@@ -2,32 +2,20 @@ trigger:
 - master
 
 jobs:
```

Comment:
Looks like you have some merge conflicts to handle which should get this file updated to what is currently committed in the repo. As it stands you are reverting some changes that were committed because your branch is behind master.

Reply:
Conflicts resolved